libs/components/list-builder/src/lib/modules/list-filters/filter.model.ts

```diff
@@ -2,36 +2,36 @@ import { ListItemModel } from '@skyux/list-builder-common';
 
 export class ListFilterModel {
   /**
-   * Specifies the name of the filter.
+   * The name of the filter.
    * @required
    */
   public name: string;
 
   /**
-   * Specifies a label for the filter's summary item.
+   * The label for the filter's summary item.
    * The label defaults to the `value` of the filter.
    */
   public label: string;
 
   /**
-   * Indicates whether users can dismiss the filter's summary item.
+   * Whether users can dismiss the filter's summary item.
    * @default true
    */
   public dismissible = true;
 
   /**
-   * Specifies the current value of the filter.
+   * The current value of the filter.
    */
   public value: any;
 
   /**
-   * Specifies the default value of the filter. When users do not set the
+   * The default value of the filter. When users do not set the
```

Comment:
@johnhwhite, do you know if the second sentence is correct? I'm not sure what it means for the users to "set the default value" since a default value would presumably be set by default without any user action.

Reply:
I'm still not 100 percent clear on what's going here, but I'd go ahead and commit this edit and move on because this change clears up the confusion around the user setting a default and because lists are deprecated, so we shouldn't hold things up any further for this property description.

(FWIW, my confusion is around saying that the default value does not affect the list. If that's the case, then what is the filter for? I assumed that the default value of the filter was something provided to pre-filter the list as it loads, which would affect the list. And if the list isn't pre-filtered based on the default value, then the wording here seems to indicate that the default value becomes a value that users can't filter on because the list isn't pre-filtered and because if they enter filter criteria that match the default, then the list will be unaffected. I might just be confusing myself, but I think something is off on the description here. ... But like I said, I think we should commit this as is and not worry about it because this is a deprecated component and because this only came up in our review of the docs, not from feedback from consumers.)

Reply:
I don't have the specific business case, and the "feature" is only used if a default value is set. It could be used by setting the default value to "everything" — a value that means no filter should be applied.